api/src/main/scala/hmda/api/http/public/InstitutionSearchPaths.scala

```diff
@@ -51,7 +52,29 @@ trait InstitutionSearchPaths extends InstitutionSearchProtocol with HmdaCustomDi
           }
         }
       }
-    }
+    } ~
+      path("institutions" / Segment) { institutionId =>
+        encodeResponse {
+          timedGet { uri =>
+            val institutionF = for {
+              a <- institutionPersistenceF
+              o <- (a ? GetInstitutionById(institutionId)).mapTo[Option[Institution]]
+              i = o.getOrElse(Institution.empty)
+            } yield i
+
+            onComplete(institutionF) {
```

Comment:
Why not this?
```scala
      path("institutions" / Segment) { institutionId =>
        encodeResponse {
          timedGet { uri =>
            val institutionF = for {
              a <- institutionPersistenceF
              o <- (a ? GetInstitutionById(institutionId)).mapTo[Option[Institution]]
            } yield o

            onComplete(institutionF) {
              case Success(Some(i)) =>
                complete(ToResponseMarshallable(i))
              case Success(None) =>
                complete(ToResponseMarshallable(HttpResponse(StatusCodes.NotFound)))
              case Failure(e) =>
                complete(ToResponseMarshallable(HttpResponse(StatusCodes.InternalServerError)))
            }
          }
        }
      }
```
Using `getOrElse(Institution.empty)` is a code smell. We may as well avoid it when it's easy to do so.